src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/ConfigurationPathTests.cs

```diff
@@ -44,6 +44,7 @@ public void CustomAppConfigIsUsedWhenSpecifiedAsAbsolutePath()
         }
 
         [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Fails when path contains #")]
```

Comment:
Should we use other prefix instead (TMP_)?

Reply:
I chose # as it only makes the paths 1 chat longer. We have had length problems in the past - hence the code in FileCleanupTestBase that shortens paths. Plus, nobody else seems to use #..